### ISSUE TITLE
Bad file path parsing (Linux)

### ISSUE THREAD

**Issue by bbusr** [bleachbit/bleachbit#205]
In Edit -> Preferences -> Personnalized -> Add a file, I add the file :

/var/log/samba/log.%m 

to be erased (bleachbit ran as root), and close the window.
If after that I click on Edit -> Preferences, nothing happens (no windows appears).
In bleachbit.ini, instead of seeing :

[custom/paths]
0_type = file
0_path = /var/log/samba/log.%m

I see :

[custom/paths]
0_type = file

(the path line is missing).
If I delete this section (in the ini) and re-run bleachbit, everything is OK.
I thing the problem comes from the "%" in the file name (log.%m)

Notes :

- This happens on bleachbit 1.12 and on the last version on github 
  (downloaded here : https://github.com/bleachbit/bleachbit ).
- If I create the file /var/log/samba/log.m (without the "%") and add it to bleachbit,
  everything works fine.

- I tested with some dummy files : a%.ext, %a.ext and a.ext% and the same bug happens.

**Comment by az0** [comment]
I confirm in BleachBit 2.0

````
Traceback (most recent call last):
  File "/software/bleachbit/bleachbit/GuiPreferences.py", line 363, in add_custom_file_cb
    options.set_custom_paths(pathnames)
  File "/software/bleachbit/bleachbit/Options.py", line 286, in set_custom_paths
    self.config.set(section, str(counter) + '_path', value[1])
  File "/usr/lib/python2.7/ConfigParser.py", line 752, in set
    "position %d" % (value, tmp_value.find('%')))
ValueError: invalid interpolation syntax in '/tmp/log%s' at position 15
````

**Comment by az0** [comment]
Thank you for the clear bug report. It is fixed. You can wait for the next release, or you can apply the simple, one-line patch to your Options.py file.

**Comment by az0** [comment]
The fix was published in [BleachBit 2.1 beta](https://www.bleachbit.org/news/bleachbit-21-beta). Please try it out, and let me know how it goes.
